src/units.js

```diff
@@ -0,0 +1,74 @@
+/* eslint no-console:0 */
+
+/**
+ * This file does conversion between units.  In particular, it provides
+ * calculateSize to convert other units into ems.
+ */
+
+const ParseError = require("./ParseError");
+
+// This table gives the number of pts in one of each *absolute* TeX unit.
+// Thus, multiplying a length by this number converts the length from units
+// into pts.  Dividing the result by ptPerEm gives the number of ems
+// *assuming* a font size of ptPerEm (normal size, normal style).
+const ptPerUnit = {
+    // https://en.wikibooks.org/wiki/LaTeX/Lengths and
+    // https://tex.stackexchange.com/a/8263
+    "pt": 1,            // TeX point
+    "mm": 7227 / 2540,  // millimeter
+    "cm": 7227 / 254,   // centimeter
+    "in": 72.27,        // inch
+    "bp": 803 / 800,    // big (PostScript) points
+    "pc": 12,           // pica
+    "dd": 1238 / 1157,  // didôt
```

Comment:
They're from https://en.wikibooks.org/wiki/LaTeX/Lengths .  What do you mean by "originals"?